crates/astria-bridge-withdrawer/src/bridge_withdrawer/submitter/mod.rs

```diff
@@ -293,11 +297,12 @@ async fn submit_tx(
                 async move {}
             },
         );
+    let tx_bytes = tx.to_raw().encode_to_vec();
     let check_tx = tryhard::retry_fn(|| {
         let client = client.clone();
-        let tx = tx.clone();
+        let tx_bytes = tx_bytes.clone();
         let span = info_span!(parent: span.clone(), "attempt send");
-        async move { client.submit_transaction_sync(tx).await }.instrument(span)
+        async move { client.broadcast_tx_sync(tx_bytes).await }.instrument(span)
```

Comment:
It's a shame that this requires `Vec<u8>` instead of `Bytes`, which would make the clone above much cheaper.